languages/python/exercises/concept/tuples/.meta/example.py

```diff
@@ -0,0 +1,21 @@
+
+def get_coordinate(record):
+    return record[1]
+
+def convert_coordinate(coordinate):
+    # alt return coordinate[0], coordinate[1]
+    return tuple(coordinate)
```

Comment:
Both are idiomatic (_for better or worse_).  But since we're trying to get the student to practice different ways of making tuples, I'll re-read in the am and make sure I have one task that uses one method, and another that uses the constructor.

Reply:
`return (coordinate[0], coordinate[1])` seems most natural to me

Reply:
> I'll re-read in the am and make sure I have one task that uses one method, and another that uses the constructor.

Brilliant!